test/cache.js

```diff
@@ -1,21 +1,19 @@
 import test from 'ava';
 import delay from 'delay';
+import {alfy} from './fixtures/utils';
```

Comment:
It's not really a fixture though. I would name it `'./fixtures/utils'` => `'./_util'`